nservicebus/operations/metrics-service-control.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Metrics data to ServiceControl
+summary: Reporting metrics to ServiceControl.
+reviewed: 2017-09-26
+component: MetricsServiceControl
+related:
+ - nservicebus/operations
+related:
+ - samples/logging/metrics
+---
+
+
+This component enables sending monitoring data gathered with NServiceBus.Metrics to an instance of ServiceControl.Monitoring service.
+
+## Enabling
+
+It can be enabled via:
```

Comment:
I'd change the title from "Enabling" to "Configuration" and then:

1. Say whether it's enabled by default or not, e.g. "After installation/adding package to the solution, metrics are/won't be automatically sent to SC" or something like this.
2. Keep what you have here, i.e. "It can be enabled via: + snippet"